scan-command/src/main/java/io/ballerina/scan/internal/ProjectAnalyzer.java

```diff
@@ -51,4 +65,51 @@ private Consumer<DocumentId> analyzeDocument(Module module, ScannerContextImpl s
             analyzer.analyze();
         };
     }
+
+    List<Issue> runExternalAnalyzers(Project project) {
+        StringBuilder newImports = new StringBuilder();
+        StringBuilder tomlDependencies = new StringBuilder();
+        int importCounter = 0;
+
+        for (ScanTomlFile.Analyzer analyzer : scanTomlFile.getAnalyzers()) {
+            String reportingSource = analyzer.getOrg() + FORWARD_SLASH + analyzer.getName();
+            String analyzerImport = IMPORT_PREFIX + reportingSource + USE_IMPORT_AS_SERVICE;
+            newImports.append(analyzerImport).append("\n");
```

Comment:
Do we need the separate `analyzerImport` variable? Why have a separate concatenation anyway? 

Please check and remove unnecessary variables, unless they help with readability.